### PR TITLE
add-possibility-to-do-cmd-click-on-symbols

### PR DESCRIPTION
**Pull request by jecisc**
Add possibility to do cmd + (shift +) click on symbols. 

Currently, when you do cmd + (shift +) click on a class or method, you can browse the class, its references, the implementors or the senders. 

With this change, when you do:
- cmd + click on a symbol representing a class, then it browse it. 
- cmd + shitf + click on a symbol representing a class, then it browse its references. 
- cmd + click on a symbol representing a selector, then it browse its implementors. 
- cmd + shift + click on a symbol representing a selector, then it browse its senders.

### CODE REVIEW

**Comment by pavel-krivanek** [comment]
The links should not store the reference to classes but just names to avoid leaks. Is it like that?

**Comment by pavel-krivanek** [comment]
yes, it is :-)

File: src/Shout/SHRBTextStyler.class.st [hunk 1]
```diff
@@ -1144,12 +1144,22 @@ at once, but for ordinary literal arrays, style every node"
...
+	- In case it is another literal value, we return a TextClassLink to browse the class defining this litera."
+	link := value isSymbol
+		ifTrue: [
+			self class environment
+				at: value
+				ifPresent: [ :class | TextClassLink class: class ]
+				ifAbsent: [ TextMethodLink selector: value ] ]
+		ifFalse: [ TextClassLink class: value class ].
+	self addStyle: (self literalStyleSymbol: value) attribute: link forNode: aLiteralValueNode
 ]
```
Comment (guillep) [inline src/Shout/SHRBTextStyler.class.st:1163]: Good this is done at the level of the syntax highlighter, thanks ^^.

**Review by guillep — APPROVED** [review]

**Review by MarcusDenker — APPROVED** [review]

**Comment by jecisc** [comment]
Next step is to make it work on Windows and Linux

**Comment by MarcusDenker** [comment]
Note: CI did not run. Strange

**Comment by MarcusDenker** [comment]
CI is ok

**Pull request merged**